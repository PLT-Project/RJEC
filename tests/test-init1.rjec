func main() {
  i, j := 42, 30;
  b := true;
  printi(i);
  printi(j);
  printb(b);

  if true {
      j = 60;
      printi(j);
      i := 52;
      printi(i);
  }
  
  printi(i);
  printi(j);
}