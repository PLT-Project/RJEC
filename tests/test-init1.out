42
30
true
60
52
42
60